CHANGELOG.md

```diff
@@ -46,6 +46,19 @@ and this project adheres to
   This fixes a bug where a microVM with incompatible balloon and guest memory
   size could be booted, due to the check for this condition happening after
   Firecracker's configuration was updated.
+- [#4259](https://github.com/firecracker-microvm/firecracker/pull/4259): Added a
+  double fork mechanism in the Jailer to avoid setsid failures occurred while
```

Comment:
nit: it's a matter of taste, but I'd prefer having `()` after a syscall name to make it clear that it is a function like man page does (e.g. https://man7.org/linux/man-pages/man2/setsid.2.html ).
```suggestion
  double fork mechanism in the Jailer to avoid setsid() failures occurred while
```